functest/package_test.go

```diff
@@ -49,3 +49,61 @@ func TestPackage(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestPackageDockerOptions(t *testing.T) {
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	defer cleanup()
+
+	var outBuffer bytes.Buffer
+	log := &cmd.LoggingConfig{}
+	log.InitLogging(&outBuffer, &outBuffer)
+
+	stackDir := filepath.Join(cmdtest.TestDirPath, "starter")
+	err := cmd.CopyDir(log, stackDir, sandbox.ProjectDir)
+	if err != nil {
+		t.Errorf("Problem copying %s to %s: %v", stackDir, sandbox.ProjectDir, err)
+	} else {
+		t.Logf("Copied %s to %s", stackDir, sandbox.ProjectDir)
+	}
+
+	// Because the 'starter' folder has been copied, the stack.yaml file will be in the 'starter'
+	// folder within the temp directory that has been generated for sandboxing purposes, rather than
+	// the usual core temp directory
+	sandbox.ProjectDir = filepath.Join(sandbox.ProjectDir, "starter")
+
+	args := []string{"stack", "package", "--docker-options", "-q"}
```

Comment:
It could be great if we could test that the `docker` command actually gets the `-q` (quiet flag). I believe here we are just testing that the `stack package --docker-options -q` command is not falling, and not that the docker options are correctly being passed down?

Reply:
After checking the `docker_options_test`, we currently only run the Appsody commands. This could be a future improvement.